devVerifiers.md

```diff
@@ -40,33 +40,45 @@ await CredentialHandlerPolyfill.loadOnce();
+#### 1. Make a Verifiable Presentation Request
+A verifier can request credentials over CHAPI by forming a *[Verifiable Presentation Request](https://w3c-ccg.github.io/vp-request-spec/)*.  The example below illustrates a `QueryByExample`, which specifies the type of Verifiable Credential being requested and an optional reason for the request.
 
 ```
-const credentialQuery = {
-  "web": {
-    "VerifiablePresentation": {
-            "query": [{
-                "type": "QueryByExample",
-                "credentialQuery": {
-                    "reason": "We need to see your University Degree to continue the teacher application process."
-                    "example": {
-                        "@context": [
-                            "https://w3id.org/credentials/v1",
-                            "https://www.w3.org/2018/credentials/examples/v1"
-                        ],
-                        "type": ["UniversityDegreeCredential"],
-                        "credentialSubject": {
-                            "id": "did:example:ebfeb1f712ebc6f1c276e12ec21"
-                        }
-                    }
-                }
-            }]
-        },
-        "recommendedHandlerOrigins": [
-            "https://chapi-demo-wallet.digitalbazaar.com"
-        ]
+const testVpr = {
+  "query": [{
+    "type": "QueryByExample",
+    "credentialQuery": {
+      "reason": "Please present your University Degree to continue the teacher application process.",
+      "example": {
+        "@context": [
+          "https://w3id.org/credentials/v1",
+          "https://www.w3.org/2018/credentials/examples/v1"
+        ],
+        "type": ["UniversityDegreeCredential"],
+        "credentialSubject": {
+          "id": "did:example:ebfeb1f712ebc6f1c276e12ec21"
+        }
+      }
     }
+  }]
+}
+```
+<p class="code-annotation">
+  <a href="https://github.com/credential-handler/chapi-demo-verifier/blob/master/index.html"
+  target="_blank" rel="noopener noreferrer"> chapi-demo-verifier/index.html </a>
+</p>
+
+#### 2. Wrap the Request in a Web Credential Query
+Next, construct a generalized *[Credential Interface Query](https://www.w3.org/TR/credential-management-1/)* of type `web`.  This will be passed through the browser to a CHAPI Credential Handler, which recognizes generalized *Web Credentials*. The example below illustrates a  Web Credential request for a `VerifiablePresentation` which uses the specific query parameters defined above.  The `recommendedHandlerOrigins` parameter allows verifiers to suggest Credential Handlers (e.g. digital wallets) for the user to present the requested data. 
```

Comment:
```suggestion
Next, construct a generalized *[Credential Interface Query](https://www.w3.org/TR/credential-management-1/)* of type `web`.  This will be passed through the browser to a CHAPI Credential Handler, which recognizes generalized *Web Credentials*. The example below illustrates a Web Credential request for a `VerifiablePresentation` which uses the specific query parameters defined above.  The `recommendedHandlerOrigins` parameter allows verifiers to suggest Credential Handlers (e.g. digital wallets) for the user to present the requested data. 
```